c2corg_ui/templates/waypoint/helpers/create_edit_summary.html

```diff
@@ -0,0 +1,288 @@
+
```

Comment:
This file contains no helper functions. It should be moved to the root of ``c2corg_ui/templates/waypoint/``.